### ISSUE TITLE
Studio creates more than one 'camunda:Properties' per element

### ISSUE THREAD

**Issue by LeoTT** [process-engine/bpmn-studio#490]
#### Describe your issue

Studio creates more than one `camunda:Properties` per element. In fact, it creates so many `properties`-elements without users knowledge or input that it makes up a big part of the resulting xml.
Here's an example, see the next hundred lines:
https://github.com/process-engine/example_processes/blob/dd288d99fd59ae9d5efe88e7148cfdbb4d40424e/solutions/Bookstore/diagrams/Checkout.bpmn#L91

#### How can others reproduce the issue?

Selection of elements in diagram creates the new properties-element (I think only once per element per session, not sure).
Select elements, count the numbers in xml-view.

Have not noticed negative effects yet, but it could lead to errors in future. Would for example fail xsd-validation.

#### Possible solution
Technically this line is the culprit:
https://github.com/process-engine/bpmn-studio/blob/75c54a94117e4f0ef460129bc2c53992067cc95f/src/modules/property-panel/indextabs/extensions/sections/basics/basics.ts#L111

Would not focus on that line and refactor a bit more instead.

- the function`_getPropertyElement` creates stuff. Don't have `get` functions create stuff.
- it's starting off by creating an `extensionselement` and inner `properties` element at the beginning. Why? Those 2 wrapping elements are only wanted when there is a `property`.

This is how it's supposed to look like with one property:
```xml
    <bpmn:startEvent id="StartEvent_1">
      <bpmn:extensionElements>
        <camunda:properties>
          <camunda:property name="my_prop" />
        </camunda:properties>
      </bpmn:extensionElements>
    </bpmn:startEvent>
```

Here no property:
```xml
    <bpmn:startEvent id="StartEvent_1" />
```
That's how the camunda modeler does it.

In short: don't initialize `extensionElements` and `properties` by default. Instead add them when needed, remove them when no longer needed.

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one
